doc/usermanual/darkroom/modules/color/lut3d.xml

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE sect3 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+		<!ENTITY % darktable_dtd SYSTEM "../../../dtd/darktable.dtd">
+		%darktable_dtd;
+		]>
+<sect3 status="draft" id="lut3d">
+
+  <title>3D LUT</title>
+
+  <indexterm>
+    <primary>modules</primary>
+    <secondary>lut3d</secondary>
+  </indexterm>
+
+  <indexterm>
+    <primary>lut3d</primary>
+  </indexterm>
+
+  <sect4>
+
+    <title>Overview</title>
+
+    <informaltable frame="none">
+      <tgroup cols="2" colsep="0" rowsep="0">
+        <colspec colwidth="6*"/>
+        <colspec colwidth="4*"/>
+        <tbody>
+          <row>
+            <entry>
+              <para>
+                A <emphasis>3D LUT</emphasis> is a tridimensional table which allows to
+                transform any RGB value into another RGB value, normaly used for film
+                simulation and color grading.
+              </para>
+              <para>
+                The module accepts <emphasis>.cube</emphasis> and <emphasis>.png (haldclut)</emphasis> files.
+                The 3D LUT data are not saved in database nor in the XMP file, but just the
+                path inside the 3D LUT folder (see below).
+              </para>
+            </entry>
+            <entry>
+              <graphic fileref="darkroom/modules/images/lut3d-1.png" scalefit="1" width="80%" align="center" />
+            </entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>
+      The module lut3d needs to find the 3D LUT file at the same place in your 3D LUT folder
```

Comment:
I'd go for "The lut3d module", but that's definitely one of the things about which I'm unsure in English. Bruce? :D